persistence/sql/persister_oauth2.go

```diff
@@ -374,35 +374,27 @@ func (p *Persister) flushInactiveTokens(ctx context.Context, notAfter time.Time,
 		notAfter = requestMaxExpire
 	}
 
-	signatures := []string{}
-
-	// Select tokens' signatures with limit
-	q := p.Connection(ctx).RawQuery(
-		fmt.Sprintf("SELECT signature FROM %s WHERE requested_at < ? AND nid = ? ORDER BY signature LIMIT %d",
-			OAuth2RequestSQL{Table: table}.TableName(), limit),
-		notAfter,
-		p.NetworkID(ctx),
-	)
-	if err := q.All(&signatures); err == sql.ErrNoRows {
-		return errorsx.WithStack(fosite.ErrNotFound)
-	} else if err != nil {
-		return errorsx.WithStack(err)
-	}
-
-	// Delete tokens in batch
 	var err error
-	for i := 0; i < len(signatures); i += batchSize {
-		j := i + batchSize
-		if j > len(signatures) {
-			j = len(signatures)
-		}
+	signature_len := batchSize
+	flushed_records := 0
+	for flushed_records < limit && signature_len == batchSize {
+		// Delete in batches
+		signature_len, err = p.Connection(ctx).RawQuery(
```

Comment:
Please use `camelCase` :)

https://betterprogramming.pub/naming-conventions-in-go-short-but-descriptive-1fa7c6d2f32a?gi=f4d3782aaa3f

Reply:
got it :)